src/main/scala/mesosphere/marathon/core/deployment/ScalingProposition.scala

```diff
@@ -19,6 +19,10 @@ object ScalingProposition extends StrictLogging {
     killSelection: KillSelection,
     runSpecId: AbsolutePathId): ScalingProposition = {
 
+    val instancesRunning: scala.collection.immutable.Seq[mesosphere.marathon.core.instance.Instance] = instances
+      .filter(_.tasksMap.size != 0)
```

Comment:
I'm not totally sure why this really differs from the `_.state.goal` as in both cases my understanding is that we are looking at instances in Marathon state. In second case they are not necessarily running yet but I don't really understand how this makes the difference...